### PR TITLE
[Security] Remove unused code about ROLE_PREVIOUS_ADMIN

### PR DESCRIPTION
**Pull request by lyrixx**
| Q             | A
| ------------- | ---
| Branch?       | 7.0
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       |
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Symfony/Bundle/SecurityBundle/Tests/DataCollector/SecurityDataCollectorTest.php [hunk 1]
```diff
@@ -115,7 +115,7 @@ public function testCollectSwitchUserToken()
         $this->assertSame('yceruto', $collector->getImpersonatorUser());
         $this->assertSame(SwitchUserToken::class, $collector->getTokenClass()->getValue());
         $this->assertTrue($collector->supportsRoleHierarchy());
-        $this->assertSame(['ROLE_USER', 'ROLE_PREVIOUS_ADMIN'], $collector->getRoles()->getValue(true));
+        $this->assertSame(['ROLE_USER', 'ROLE_KOALA'], $collector->getRoles()->getValue(true));
```
Comment (OskarStark) [inline src/Symfony/Bundle/SecurityBundle/Tests/DataCollector/SecurityDataCollectorTest.php]: One role would be enough, wouldn't it?

Comment (lyrixx) [reply src/Symfony/Bundle/SecurityBundle/Tests/DataCollector/SecurityDataCollectorTest.php]: Don't you like koala ? 😭 

haha, ok, I'll remove it

Comment (OskarStark) [reply src/Symfony/Bundle/SecurityBundle/Tests/DataCollector/SecurityDataCollectorTest.php]: I really like it 😅😘❤️

**Review by nicolas-grekas — COMMENTED** [review]
Related to https://github.com/symfony/symfony/pull/35858
@wouterj WDYT?

**Review by nicolas-grekas — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @lyrixx.

**Pull request merged**